html/input_validator_example.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="EN">
+<head>
+    <meta charset="UTF-8">
+    <title>Example</title>
+    <link type="text/css" rel="stylesheet" href="css/style.css"/>
+    <script type="text/javascript" src="js/input_validator.js"></script>
```

Comment:
As far as I can tell this has no effect. The same is true for `jquery.validate.min.js`. You need to actually use them